Obj_To_Stl/headers/Triangle.h

```diff
@@ -0,0 +1,28 @@
+#pragma once
+#include "Point3D.h"
+
+namespace Shapes3D
+{
+    class Triangle
+    {
+    public:
+        Triangle();
+        Triangle(int inIndex1, int inIndex2, int inIndex3, int inNormalIndex);
+        ~Triangle();
+
+        int index1() const;
+        int index2() const;
+        int index3() const;
+        int normalIndex() const;
+
+        void setindex1(int inIndex1);
+        void setindex2(int inIndex2);
+        void setindex3(int inIndex3);
+
+    private:
+        int mIndex1;

```

Comment:
Triangle should always store 3 Point3D vertices and 1 Point3D normal.

Reply:
Triangle class has "int" variables (we are declaring indices and so it is taken as "int") as mentioned by Vinayak. I have later mapped "Point3D" and with "int" indices in Reader file.